### ISSUE TITLE
Geopackage single table publication issue when table name differs from geopackage name

### ISSUE THREAD

**Issue by eblondel** [r-geoflow/geoflow#287]
There is a Geopackage single table publication issue when table name differs from geopackage name. In that way the `nativename` associated to the Geoserver feature type is set to the same as `source` name and causes an issue (it should be the geopackage table name).

**Comment by eblondel** [comment]
Yes true, the difficulty is that geoflow distinguishes databases / repositories vs. data files. In the case of geopackage it's a mix of information, from a user viewpoint sometimes it's considered just as standarded modern format to replace a shapefile (so 1 table basicaly), in other it's a file materialization of a complete database (which is the actual way it is designed since it handles a sqlite db behind).
The other point is also true, geopackage goes beyond the classical distinction between vector and grid.
All this deserves brainstorming to see how to best manage geopackages in geoflow, covering all possible uses.

**Comment by kikislater** [comment]
Well, a bit difficult as gpkg could also include qgis project and layers style but well documented !
